### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by apeters**
This branch adds accessibility functionality and styling to arches.  This is still a work in progress

### CODE REVIEW

File: arches/app/templates/change_password.htm [hunk 1]
```diff
@@ -1,64 +1,57 @@
...
     <div class="change-password-form panel panel-default">
-    <div class="panel-heading">{% trans "Change your password" %}</div>
-            <div class="panel-body">
-                <form action="?next={{ next }}" method='POST' {% if auth_failed %} autocomplete="off" {% endif %}>
-                    {% csrf_token %}
-                    <div class="form-group">
-
-                        <div>
-                            <input type="password" class="form-control input-lg" placeholder="Old Password" data-bind="textInput:credentials.old_password" name='old_password' autofocus>
+        <div class="panel-heading">{% trans "Change your password" %}</div>
```
Comment (chiatt) [inline arches/app/templates/change_password.htm:6]: The password reset form has more empty space that pushes it below the bottom of the page. The form inputs are no longer centered and placeholders are no longer center aligned. In these two screenshots, the shorter one is in dev/7.4.x and the taller is in the accessibility branch.

image

image

File: arches/app/templates/registration/password_reset_form.html [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-			</div>
-			<div class="arches-signin-btn">
-				<button class="btn btn-primary btn-lg btn-block" type="submit">{% trans "Recover my password" %}</button>
-			</div>
-		</div>
+    <form method="post" action="{{ url }}">
+        <div class="form-group">
 
-	</form>
+            {% csrf_token %}
```
Comment (chiatt) [inline arches/app/templates/registration/password_reset_form.html:10]: The password reset form has a lot of empty space that pushes it below the bottom of the page. 
image

image

File: arches/app/templates/login.htm [hunk 3]
```diff
@@ -26,93 +9,97 @@
...
 <div id="container" class="cls-container arches-login">
 
-	<!-- BACKGROUND IMAGE -->
-	<!--===================================================
-	<div id="bg-overlay" class="bg-img img-login"></div>
-	-->
+    <!-- BACKGROUND IMAGE -->
+    <!--===================================================
+    <div id="bg-overlay" class="bg-img img-login"></div>
+    -->
```
Comment (chiatt) [inline arches/app/templates/login.htm:15]: The login page also has changed. In the accessibility branch, the placeholder is now outside of the input and is quite small.

image

image

**Review by chiatt — COMMENTED** [review]
I still have to take a look at some more templates, but I've found a couple of changes that we may want to fix.

File: arches/app/templates/change_password.htm (hunk 1 above)
Comment (apeters) [reply arches/app/templates/change_password.htm:6]: @chiatt Yeah, this seems like a regression.

File: arches/app/templates/login.htm (hunk 3 above)
Comment (apeters) [reply arches/app/templates/login.htm:15]: Let me know what you think of these changes

File: arches/app/templates/change_password.htm (hunk 1 above)
Comment (apeters) [reply arches/app/templates/change_password.htm]: This should now be fixed

**Pull request merged**